util/gh-pages/index_template.html

```diff
@@ -50,14 +51,8 @@
             <h1>Clippy Lints</h1> {# #}
         </div> {# #}
 
-        <noscript> {# #}
-            <div class="alert alert-danger" role="alert"> {# #}
-                Sorry, this site only works with JavaScript! :( {# #}
-            </div> {# #}
-        </noscript> {# #}
```

Comment:
Let's leave some kind of message to say the filtering requires JS

Reply:
Good idea!